test/unit/specs/events/order/cd-event-sessions.spec.js

```diff
@@ -449,17 +449,36 @@ describe('Event sessions component', () => {
       });
     });
 
-    describe('methods.addNewChildren()', () => {
+    describe.only('methods.addNewChildren()', () => {
```

Comment:
You could've used the new suggestion feature 😛 

```suggestion
    describe('methods.addNewChildren()', () => {
```